aquarius/app/dao.py

```diff
@@ -84,33 +84,28 @@ def is_listed(services):
                 ):
                     return service["attributes"]["curation"]["isListed"]
 
-    def run_es_query(self, query, sort, page, offset, text=None):
+    def run_es_query(self, data):
         """do an elasticsearch native query.
 
         The query is expected to be in the elasticsearch search format.
 
         :return: list of objects that match the query.
         """
+        page = data.get("page")
         assert page >= 1, "page value %s is invalid" % page
 
-        logging.debug(f"elasticsearch::query::{query}, text {text}")
+        sort = data.get("sort")
         if sort is not None:
             self.oceandb._mapping_to_sort(sort.keys())
             sort = self.oceandb._sort_object(sort)
         else:
             sort = [{"_id": "asc"}]
 
-        if text:
-            sort = [{"_score": "desc"}] + sort
-            if isinstance(text, str):
-                text = [text]
-
-            text = [t.strip() for t in text]
-            text = [t.replace("did:op:", "0x") for t in text if t]
-
```

Comment:
is text search deprecated? does the query alone support the existing use cases in the market?

Reply:
Yes, the query can have text inside it, market asked to remove the shortcut